.travis.yml

```diff
@@ -2,25 +2,12 @@
 
 language: python
 python:
+  - 3.7
   - 3.6
   - 3.5
-  - 3.4
-  - 2.7
 
 # command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
 install: pip install -U tox-travis
 
 # command to run tests, e.g. python setup.py test
 script: tox
-
```

Comment:
we might want this for future @mbjoseph but let's get appveyor happy first. it shows people how to add auto pypi deployment which i think is really nice. but perhaps you had some thought about why we don't want it?

i had these appveyor issues too. i'm sorry i cant be more helpful as appveyor is still a black box for me. for me it was installing earthpy without deps that fixed things as once something was installed i was getting an error trying to install the same dependencies again. i'm not actually sure why.

Reply:
@lwasser the relevant deploy code for projects created using this template is elsewhere: https://github.com/pyOpenSci/cookiecutter-pyopensci/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/.travis.yml#L19-L34

Reply:
ahhh thanks @mbjoseph i should have dug a bit more deeply!!